deploy/template.yaml

```diff
@@ -1425,6 +1425,271 @@ Resources:
+                ],
+                [ "ECS/ContainerInsights", "DesiredTaskCount", "ClusterName", "${ECSCluster}",
+                  "ServiceName", "${ContainerService.Name}"
+                ],
+                [ "ECS/ContainerInsights", "PendingTaskCount", "ClusterName", "${ECSCluster}",
+                    "ServiceName", "${ContainerService.Name}"
+                  ]
+              ],
+              "view": "timeSeries",
+              "stacked": false,
+              "stat": "Sum",
+              "period": 60
+            }
+          },
+           {
+             "height": 6,
+             "width": 12,
+             "y": 0,
+             "x": 12,
+             "type": "metric",
+             "properties": {
+               "title": "ECS Cluster Service Count",
+               "region": "${AWS::Region}",
+               "metrics": [
+                 [ "ECS/ContainerInsights", "ServiceCount", "ClusterName", "${ECSCluster}" ]
+               ],
+               "view": "timeSeries",
+               "stacked": false,
+               "stat": "Average",
+               "period": 60
+             }
+           },
+          {
+            "height": 6,
+            "width": 12,
+            "y": 0,
+            "x": 12,
+            "type": "metric",
+            "properties": {
+              "title": "Number Of Requests Made To ELB vs Response Time",
+              "region": "${AWS::Region}",
+              "metrics": [
+                [ "AWS/ApplicationELB", "RequestCount", "LoadBalancer", "${ApplicationLoadBalancer.LoadBalancerFullName}", { "yAxis": "right" } ],
+                [ "AWS/ApplicationELB", "TargetResponseTime", "LoadBalancer", "${ApplicationLoadBalancer.LoadBalancerFullName}",
+                  {
+                    "stat": "Average"
+                  }
+                ]
+              ],
+              "view": "timeSeries",
+              "stacked": false,
+              "stat": "Sum",
+              "period": 60
+            }
+          },
+          {
+            "height": 6,
+            "width": 12,
+            "y": 0,
+            "x": 12,
+            "type": "metric",
+            "properties": {
+              "title": "Load Balancer Endpoint Functionality",
+              "region": "${AWS::Region}",
+              "metrics": [
+                [ "AWS/ApplicationELB", "ActiveConnectionCount", "LoadBalancer", "${ApplicationLoadBalancer.LoadBalancerFullName}",{ "yAxis": "right" } ],
+                [ "AWS/ApplicationELB", "ProcessedBytes", "LoadBalancer", "${ApplicationLoadBalancer.LoadBalancerFullName}" ]
+              ],
+              "view": "timeSeries",
+              "stacked": false,
+              "stat": "Sum",
+              "period": 60
+            }
+          },
+          {
+            "height": 6,
+            "width": 12,
+            "y": 0,
+            "x": 12,
+            "type": "metric",
+            "properties": {
+              "title": "ELB 502,503 And 504 Errors",
```

Comment:
nit: ALB as it's an application load balancer

Reply:
Edited now thanks!